tests-e2e/argocd/queryscoped_cluster_secrets_test.go

```diff
@@ -215,26 +195,9 @@ var _ = Describe("Argo CD Application tests", func() {
 
 				By("creating an Argo CD application to deploy to the user's namespace")
 
-				argoCDApplication := appv1alpha1.Application{
-					ObjectMeta: metav1.ObjectMeta{
-						Name:       "test-" + userName,
-						Namespace:  argoCDNamespace,
-						Finalizers: []string{"resources-finalizer.argocd.argoproj.io"},
-					},
-					Spec: appv1alpha1.ApplicationSpec{
-						Source: appv1alpha1.ApplicationSource{
-							RepoURL: "https://github.com/redhat-appstudio/managed-gitops/",
-							Path:    "resources/test-data/sample-gitops-repository/environments/overlays/dev",
-						},
-						Destination: appv1alpha1.ApplicationDestination{
-							Name:      "test-" + userName,
-							Namespace: otherUser, // Attempt to deploy into a Namespace we don't have access to
-						},
-						SyncPolicy: &appv1alpha1.SyncPolicy{
-							Automated: &appv1alpha1.SyncPolicyAutomated{Prune: true, SelfHeal: true},
-						},
-					},
-				}
+				// Attempt to deploy into a Namespace we don't have access to
```

Comment:
As above, for the calls to BuildArgoCDApplication, IMHO it's a bit more elegant in this case to leave it as a Go obj.